src/scenes/modlist.lua

```diff
@@ -157,61 +185,161 @@ end
 
 -- gives the text for a given mod
 local function getLabelTextFor(info)
-    local grey = { 1, 1, 1, 0.5 }
-    local white = { 1, 1, 1, 1 }
+    -- colors calculated from https://learn.microsoft.com/en-us/dotnet/api/system.windows.media.colors
+    local normalColor = { 1, 1, 1, 1 } -- white
+    local disabledColor = { 1, 1, 1, 0.5 } -- white but half-transparent
+    local favoriteColor = { 1, 0.0784313725, 0.5764705882, 1 } -- deep pink
+    local dependencyColor = { 0.8549019608, 0.6470588235, 0.1254901961, 1 } -- goldenrod
+    local dependencyOfFavoriteColor = { 1, 0.7137254902, 0.7568627451, 1 } -- light pink
```

Comment:
the contrast is a bit dubious in light mode :sweat_smile: 

<img width="1100" height="628" alt="image" src="https://github.com/user-attachments/assets/4acf4650-8495-4297-bae6-278c905d7d8a" />